### ISSUE TITLE
Any alignment test

### ISSUE THREAD

**Issue by Hannibal046** [lucadiliello/bleurt-pytorch#1]
Hi, thanks sooo much for this great work. I am looking for something like this for a while and this really helps. 

I am wondering have you ever conducted some aligment test to match the performance of original bleurt implementation by google?

**Comment by lucadiliello** [comment]
Hello @Hannibal046 and thanks for your interest! The results should be in line with both BLEURT metric on HuggingFace (since the weights are the same) and also with the original Google repository. However, the latter is in `tensorflow`, so there may be some slightly differences. I didn't do any alignment tests yet, however this is a good idea to improve this library. I will leave the issue open till I will have some time to do the test.

**Comment by SefaZeng** [comment]
I use both original BLEURT models and bleurt-pytorch to calculate scores in the same 10 pairs, the maximum difference in score for a single pair is 0.14% and the average is 0.06%.
Looks good to me.
